GET2019-E19-1219/src/Assignment1/Product.java

```diff
@@ -0,0 +1,53 @@
+package Assignment1;
+

```

Comment:
Add comments on for class to describe class functionality.

Reply:
ok sir i will make sure to add comments whenever it is required